src/Components/Rockets.js

```diff
@@ -27,7 +32,7 @@ const Rockets = () => {
                 type="button"
                 onClick={() => handleReserveRocket(rocket.id)}
               >
-                Reserve Rocket
+                {rocket.reserved ? 'Cancel Reservation' : 'Reserve Rocket'}
```

Comment:
Great work on the buttons, but I'm afraid there is an Icon just before the description that indicates that the rocket has been reserved, I suggest you  add this as it  will improve the user experience and easy accessibility to know reserved rockets

- check the original design below

![correct](https://github.com/tomdan-ai/Travalers-space-hub/assets/88054834/77777e44-d9a2-464d-a279-98d5f1ccfa16)